contributors/devel/help-wanted.md

```diff
@@ -2,12 +2,91 @@
+  The task is agreed upon and does not require further discussions
+  in the community. Call out if that area of code is untested and requires new fixtures.
+
+  API / CLI behavior is decided and included in the OP issue, for example: _"The new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`"_, with expected validations called out.
+
+- **Goldilocks priority**<br/>
+  Not too high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
+
+- **Up-To-Date**<br/>
+  Often these issues become obsolete and have already been done, are no longer desired,
+  no longer make sense, have changed priority or difficulty , etc.
 
 Related commands:
 
 - `/help` : adds the `help wanted` label to an issue
-- `/remove-help` : removes the `help wanted` label from an issue
\ No newline at end of file
+- `/remove-help` : removes the `help wanted` label from an issue
+
+# Good First Issue
+
+Items marked with the `good first issue` label are intended for _new contributors_.
+These make it clear to new contributors that they are welcome, valued, and provide
+a clear starting point for joining the project. After a contributor has successfully
+completed 1-2 `good first issue`'s, they should be ready to move on to `help wanted` items, saving remaining `good first issue`'s for other new contributors.
+
+Please keep an eye out for pull requests from new contributors and help
+move it through our processes. New contributors shouldn't be left to decipher prow
+commands, build flakes, find an approver, or ping for reviews.
+
+These items need to ensure that they follow the guidelines for `help wanted` labels (above)
+in addition to meeting the following criteria:
+
+- **No Barrier to Entry**<br/>
+  The task is something that a new contributor can tackle without
+  advanced setup, or domain knowledge.
+
+- **Solution Explained**<br/>
+  The recommended solution is clearly described in the issue.
+
+- **Provides Context**<br/>
+  If background knowledge is required, this should be explicitly mentioned
+  and a list of suggested readings included.
+
+- **Gives Examples**<br/>
+  Link to examples of similar implementations so new contributors have a reference guide
+  for their changes.
+
+- **Identifies Relevant Code**<br/>
+  The relevant code and tests should be linked in the issue.
+
+- **Ready to Test**<br/>
+  There should be existing tests that can be modified, or existing test
+  cases fit to be copied. If the area of code doesn't have tests, before labeling the issue,
+  add a test fixture. This prep often makes a great `help wanted` task!
+
+Related commands:
+
+- `/good-first-issue` : adds the `good first issue` label to an issue
+- `/remove-good-first-issue` : removes the `good first issue` label from an issue
+
+## Suggestions
+
+Provide extra assistance during reviews on `good first issue` pull requests:
```

Comment:
I think this is somewhat covered in suggestions but can we explicitly mention that someone must be ready to shepherd the PR? I have come across several situations where the new contributor creates the PR with excitement, someone reviews it but gets stuck with waiting for `/lgtm` and/or `/approve`.

I have had contributors reach out to me in DMs asking me how they could get their PR to move forward. I end up pinging the approver/reviewer on Slack or ask them to do so.

The excitement usually dies if the PR ends up waiting for approval. :(

Reply:
+1 to this - help the new contributor haul in an approver.

Reply:
Agreed, I saw in the original issue that it's a burden to provide mentorship for good first issues, but really ... this doesn't work well without it. Because as you've said trying to get a first PR approved is daunting and often turns people off of it all.